packages/react-components/react-tabs/library/src/components/InteractiveTab/InteractiveTab.types.ts

```diff
@@ -0,0 +1,34 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import type { TabSlots, TabInternalProps, TabInternalState } from '../Tab/Tab.types';
+
+export type InteractiveTabSlots = {
+  /**
+   * Root of the component.
+   */
+  root: Slot<'div'>;
+
+  /**
+   * button
+   */
+  button: NonNullable<TabSlots['root']>;
+
+  contentBefore?: Slot<'span'>;
+
+  contentAfter?: Slot<'span'>;
+} & Omit<TabSlots, 'root'>;
+
+export type InteractiveTabProps = Omit<ComponentProps<Partial<InteractiveTabSlots>, 'button'>, 'content'> &
```

Comment:
I don't understanding omitting button and content, then adding back content.

Bernardo changed the TabProps back in January to something that doesn't make sense to me either:

export type TabProps = Omit<ComponentProps<Partial<TabSlots>>, 'content'> &
  Pick<Partial<TabSlots>, 'content'> &

Can you chat with him about what is going on here?

Reply:
It was copied over from the Tab component, I simplified the type